grammar-error-corrector/data_curation.ipynb

```diff
@@ -60,37 +38,83 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from gantry.curators import BalancedStratificationCurator\n",
+    "from gantry.curators import BoundedRangeCurator\n",
+    "\n",
+    "new_accounts_curator = BoundedRangeCurator(\n",
+    "    name=f\"{GantryConfig.GANTRY_APP_NAME}-new-accounts-curator\",\n",
+    "    application_name=GantryConfig.GANTRY_APP_NAME,\n",
+    "    limit=1000,\n",
+    "    bound_field=\"inputs.account_age_days\",\n",
+    "    lower_bound=0,\n",
+    "    upper_bound=5,\n",
     "\n",
-    "bsc = BalancedStratificationCurator(\n",
-    "    name = \"\",\n",
-    "    application_name = \"\",\n",
-    "    stratify_on = \"\",\n",
-    "    limit = 10000, \n",
     ")\n",
     "\n",
-    "bsc.create()"
+    "new_accounts_curator.create()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for curator in gantry.curators.get_all_curators(GantryConfig.GANTRY_APP_NAME):\n",
+    "    print(curator.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "curator = gantry.curators.get_all_curators(GantryConfig.GANTRY_APP_NAME)[0]"
```

Comment:
@oscarbatori this would be better if you just did `dataset = new_accounts_curator.get_curated_dataset()` plz change